docs/source/estimator_overview.rst

```diff
@@ -0,0 +1,16 @@
+.. _estimator_overview:
+
+==================
+Estimator Overview
+==================
+
+Welcome to sktime's software development plan.
+
+.. note::
+
+    The estimator overview is under development. We have created a basic
+    structure and are looking for further improvements to develop the guide
+    better which can include status, references, etc.
```

Comment:
We can remove these lines here

Reply:
Yes fine, we can. Or we can edit it by adding something in the note section that might be good for reflecting upon what it does.